.github/workflows/llvm-ci.yml

```diff
@@ -0,0 +1,254 @@
+        id: prefix
+        run: |
+          echo "prefix=${RUNNER_TEMP}/llvm-install" >> "$GITHUB_OUTPUT"
+
+      # ---------- Build *nix ----------
+      - name: Build (Linux & macOS)
+        if: runner.os != 'Windows'
+        env:
+          CMAKE_INSTALL_PREFIX: ${{ steps.prefix.outputs.prefix }}
+        run: |
+          echo "Using LLVM_TAG=${LLVM_TAG}"
+
+          mkdir -p llvm-project
+          cd llvm-project
+          git init
+          git remote add origin https://github.com/llvm/llvm-project.git
+          git fetch --depth 1 origin "${LLVM_TAG}"
+          git checkout --detach FETCH_HEAD
+
+          cmake -S llvm -B build -G Ninja \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_INSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}" \
+            -DLLVM_ENABLE_PROJECTS="clang;lld" \
+            -DLLVM_ENABLE_RTTI:BOOL=ON \
+            -DLLVM_ENABLE_LIBXML2=OFF \
+            -DLLVM_ENABLE_ZLIB=OFF \
+            -DLLVM_ENABLE_ZSTD=OFF \
+            -DLLVM_TARGETS_TO_BUILD="host;NVPTX;AMDGPU" \
+            -DLLVM_ENABLE_TERMINFO=OFF
+
+          cmake --build build --target all -- -k 0
+          cmake --build build --target install
+
+      # ---------- Build Windows ----------
+      - name: Build (Windows)
+        if: runner.os == 'Windows'
+        shell: pwsh
+        env:
+          CMAKE_INSTALL_PREFIX: ${{ steps.prefix.outputs.prefix }}
+        run: |
+          $ErrorActionPreference = 'Stop'
+          Write-Host "Using LLVM_TAG=$env:LLVM_TAG"
+
+          $vsPath = "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\VsDevCmd.bat"
+          cmd /c "`"$vsPath`" && set" | ForEach-Object {
+              if ($_ -match '^([^=]+)=(.*)') {
+                  [Environment]::SetEnvironmentVariable($matches[1], $matches[2])
+              }
+          }
+
+          New-Item -ItemType Directory -Force -Path llvm-project | Out-Null
+          Set-Location llvm-project
+          git init
+          git remote add origin https://github.com/llvm/llvm-project.git
+          git fetch --depth 1 origin "$env:LLVM_TAG"
+          git checkout --detach FETCH_HEAD
+
+          cmake -S llvm -B build -G "Visual Studio 17 2022" -A x64 `
+            -DCMAKE_BUILD_TYPE=Release `
+            -DCMAKE_INSTALL_PREFIX="$env:CMAKE_INSTALL_PREFIX" `
+            -DLLVM_ENABLE_PROJECTS="clang;lld" `
+            -DLLVM_ENABLE_RTTI:BOOL=ON `
+            -DLLVM_ENABLE_LIBXML2=OFF `
+            -DLLVM_ENABLE_ZLIB=OFF `
+            -DLLVM_ENABLE_ZSTD=OFF `
+            -DLLVM_TARGETS_TO_BUILD="host;NVPTX;AMDGPU" `
+            -DLLVM_ENABLE_TERMINFO=OFF `
+            -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreadedDLL"
+
+          cmake --build build --config Release --target ALL_BUILD
+          cmake --build build --config Release --target INSTALL
+
+      # ----------  compute package naming ----------
+      - name: Compute package name
+        id: pkgmeta
+        run: |
+          os="${RUNNER_OS}"
```

Comment:
@johnnynunez Where is RUNNER_OS defined?

Reply:
Oh I see, it's a standard github var:
<img width="1680" height="1098" alt="Screenshot 2025-09-26 at 5 08 25 AM" src="https://github.com/user-attachments/assets/9409da81-9798-463f-987d-630b11511823" />